### ISSUE TITLE
Client side stream hangs in version 1.7.2

### ISSUE THREAD

**Issue by agourlay** [hseeberger/akka-sse#98]
Since I have updated my application to akka-sse 1.7.2, my client side stream hangs and does not produce any elements.

I was able to reproduce the error using the examples provided in the readme.

```scala
 Source.single(request)
      .via(Http().outgoingConnection("localhost", 8080))
      .mapAsync(1)(Unmarshal(_).to[Source[ServerSentEvent, Any]])
      .runForeach(_.runForeach(event ⇒ println(event)))
```

Using my API written with akka-http, this piece of code prints properly events with akka-sse 1.7.1 but not in 1.7.2. I did check that the API works properly.

I guess it is a change introduced in Akka-http 2.4.3 as no other changes were shipped in 1.7.2.

Thank you for this nice library :)

**Comment by huntc** [comment]
I'm also finding a similar outcome - I don't think it is akka-sse per se, but rather an issue with Akka 2.4.3 and persistent connections. I shall raise an issue with Akka.

**Comment by huntc** [comment]
Issue on Akka raised: https://github.com/akka/akka/issues/20270

**Comment by ktoso** [comment]
This is fixed in https://github.com/akka/akka/pull/20254 and will be part of Akka 2.4.4, thanks for reporting @agourlay

**Comment by hseeberger** [comment]
Fantastic. Thanks a lot to all involved here!

**Comment by hseeberger** [comment]
Fixed in #100.
